templates/dns_zone.py

```diff
@@ -0,0 +1,82 @@
+""" This template creates a Cloud DNS zone for Private Google Access"""
```

Comment:
File-level comment: maybe naming this private_access_dns_zone would be clearer? Not a strong feeling, but this isn't really a generic dns_zone template.

Reply:
definitely! I made added `private_google_access` to dns_zone names to make it very clear (hopefully)